README.md

```diff
@@ -1026,6 +1027,31 @@ datetime                   inExchanges  inTopHoldersTotal  outsideExchanges
 2019-04-11 00:00:00+00:00     7.975260          13.296356          5.321096
 ```
 
+## Token Top Transactions
+
+Top transactions for the token of a given project
+
+```python
+san.get(
+    "token_top_transactions/santiment",
+    from_date="2019-04-18",
+    to_date="2019-04-30",
+    limit=5,
+)
+```
+
+Example result:
+
+```
+
+datetime                                                         fromAddress                                          toAddress                                            trxHash      trxValue
+2019-04-21 13:51:59+00:00  {'address': '0x1f3df0b8390bb8e9e322972c5e75583...  {'address': '0x5eaae5e94995202c2183956369b1700...  0xdbced935b09dd0b9747d004f6242eb1e31b4770ae994...  166674.00000
```

Comment:
Something is missing from from the address here (`...`).
Also, let's map `toAddress` to just address and not the json `'adderss': <addr>, `isExchange`: <boolean>`.
What we could do is to add 2 additional columns `toAddressIsExchange` and `fromAddressIsExchange` with boolean values

Reply:
You're right, it looks better this way as you said.